docs/workflow/building/coreclr/osx-instructions.md

```diff
@@ -26,7 +26,9 @@ After the build has completed, there should some files placed in `artifacts/bin/
 Create the Core_Root
 ===================
 
-The Core_Root folder will have the built binaries, from `src/coreclr/build.sh` and it will also include the CoreFX packages required to run tests.
+The Core_Root folder will contain the built binaries, generated by `build.sh`, as well as the library packages required to run tests. It is required that you build
```

Comment:
This isn't true. You can pass the library configuration to the script using `/p:LibrariesConfiguration=<LIBS_CONFIG>` where can be `Debug` or `Release`.